### PR TITLE
#added to login navbar

### PR DESCRIPTION
**Pull request by jayu1999**
# installed npm dependecies

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -3,11 +3,16 @@
   "version": "0.1.0",
   "private": true,
   "dependencies": {
+    "@emotion/react": "^11.9.3",
+    "@emotion/styled": "^11.9.3",
+    "@mui/icons-material": "^5.8.4",
```
Comment (mahakagarwal248) [inline package.json:8]: not used.. either use it or remove it from package.json

File: src/components/Navbar.js [hunk 2]
```diff
@@ -11,17 +12,19 @@ const Navbar = () => {
...
           <li>
             <a href="">Shop</a>
           </li>
           <li>
-            <a href="">News</a>
+            <a href="">Contact</a>
           </li>
           <li>
-            <a href="">Contact</a>
+            <a href="D:\Ecommerce_Website\my-app\src\components\Login.js">Login</a>
```
Comment (mahakagarwal248) [inline src/components/Navbar.js:27]: Link tag from react-router can be used here

**Review by mahakagarwal248 — CHANGES_REQUESTED** [review]

File: src/components/Navbar.js (hunk 2 above)
Comment (jayu1999) [reply src/components/Navbar.js:27]: acknolwedge, added TODO comment

**Review by mahakagarwal248 — APPROVED** [review]
ack all changes , one change is pending as TODO

**Pull request closed without merging**